polaris-intellisense/src/data/allTokens.ts

```diff
@@ -0,0 +1,260 @@
+export const allTokens = {
+  "shadow-transparent": "0 0 0 0 transparent",
```

Comment:
need to get rid of this and just pull tokens directly from the `tokens` directory

Reply:
Agreed! Perhaps, we can create a npm script for this (e.g. `yarn get-polaris-tokens`) and run it to populate the tokens at various stages of the development/publish cycle:
- postinstall
- build
- prepublish